### DISCUSSION TITLE
ILN, Where does ddt4all get this article from?

### DISCUSSION THREAD

**Discussion by monalos** [cedricp/ddt4all#1181] [General]
Hello

Where does ddt4all get this article from? After all, someone wrote this ILN

ILN_enginecoolanttemp 

Does anyone know what ILN at this value means? I want to learn this information

![IMG_8961](https://github.com/user-attachments/assets/e8987b46-a5c2-4148-81f0-8691601a6541)

This value is not the engine water temperature in Celsius because when the engine water temperature is 60 degrees, this value is written as 114. When the engine water temperature is 90 degrees, it is written as 135. When the engine water temperature sensor is unplugged, it is written as 255.

**Comment by monalos** [comment]
@cedricp

**Comment by monalos** [comment]
@Furtif

**Comment by monalos** [comment]
@matzel82

**Comment by shrlnm** [comment]
@monalos There is only one way. Find all parameters in ddt database with prefix ILN_ and try to guess what it means. Usually such prefix means subsystem name.

**Reply by monalos** [reply]
I looked and didn't find it. can you help me? how will I do
